app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/partner/PartnerGrsController.scala

```diff
@@ -164,7 +160,10 @@ class PartnerGrsController @Inject() (
       val partnershipDetails = Some(
         PartnerPartnershipDetails(
           partnershipType = request.registration.organisationDetails.inflightPartnerType.get,
-          partnershipName = None,
+          partnershipName =
```

Comment:
This block is a placeholder explaining what we need todo about GRS not providing names for non incorporated partnerships.